### ISSUE TITLE
[🐛 Bug]: Deprecation warnings in Cucumber

### ISSUE THREAD

**Issue by grant-mccarriagher** [webdriverio/webdriverio#10408]
### Have you read the Contributing Guidelines on issues?

- [X] I have read the [Contributing Guidelines on issues](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md#reporting-new-issues).

### WebdriverIO Version

8.10.2

### Node.js Version

16.13

### Mode

WDIO Testrunner

### Which capabilities are you using?

```typescript
{
  runner: "local",
  framework: "cucumber",
  cucumberOpts: {
    requireModule: [],
    failAmbiguousDefinitions: true,
    ignoreUndefinedDefinitions: false,
    profile: ["default"],
    require: [
      "./features/support/hooks/**/*.js",
      "./features/stepDefinitions/**/*.js",
    ],
    tagExpression: "not @bugged",
    timeout: 6000000,
    colors: false
  },
  specs: ["features/**/*.feature"],
  logLevel: "error",
  deprecationWarnings: false,
  bail: 0,
  baseUrl: "http://localhost",
  waitforTimeout: 30000,
  connectionRetryTimeout: 200000,
  connectionRetryCount: 3,
  specFileRetries: process.env.RETRIES
    ? parseInt(process.env.RETRIES)
    : undefined,
  reporters: [
    "spec"
  ],
  services: ["shared-store"],
  hostname: "localhost",
  port: 4723,
  path: "/wd/hub/",
  maxInstances: 1
};
```

### What happened?

Seeing a lot of deprecation warnings related to Cucumber in the test output, which makes it hard to parse when debugging test issues.

### What is your expected behavior?

I'd expect deprecation warnings to be suppressed if log level is set to error.

### How to reproduce the bug.

I see this issue when running cucumber tests. Here's the versions of my dependencies:

```
    "@wdio/cli": "^8.10.2",
    "@wdio/config": "^8.10.2",
    "@wdio/cucumber-framework": "^8.10.3",
    "@wdio/local-runner": "^8.10.2",
    "@wdio/shared-store-service": "^8.10.2",
    "@wdio/spec-reporter": "^8.10.2",
    "webdriverio": "^8.10.2"
```

### Relevant log output

```typescript
[0-1] (node:65950) DeprecationWarning: `parseGherkinMessageStream` is deprecated, use `loadSources` instead; see https://github.com/cucumber/cucumber-js/blob/main/docs/deprecations.md
[0-1] (node:65909) DeprecationWarning: `Runtime` is deprecated, use `runCucumber` instead; see https://github.com/cucumber/cucumber-js/blob/main/docs/deprecations.md
```
```


### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

**Comment by christian-bromann** [comment]
Duplicate of #8149

Any contributions are welcome!
